src/test/playwright/support/pageobjects/exercises/quiz/QuizExerciseCreationPage.ts

```diff
@@ -46,17 +46,23 @@ export class QuizExerciseCreationPage {
         await this.page.locator('#drag-and-drop-question-title').fill(title);
 
         await this.uploadDragAndDropBackground();
-        await this.page.mouse.move(50, 50);
+        const element = await this.page.$('.background-area');
+        const boundingBox = await element?.boundingBox();
+
+        if (!boundingBox) {
+            throw new Error('Could not get bounding box of element');
+        }
```

Comment:
```suggestion
        expect(boundingBox, { message: 'Could not get bounding box of element' }).not.toBeNull();
```
Assertions with `expect()` give more details when they fail. But please note that this way `boundingBox` is not unwrapped anymore, so a few small changes might be needed in following lines.